PukingPredator/Assets/Scripts/Inventory.cs

```diff
@@ -1,57 +1,70 @@
-using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
 
 public class Inventory : MonoBehaviour
 {
-    // Stack to store items in the inventory
-    private Stack<Item> items;
-
-    // Start is called before the first frame update
-    void Start()
+    /// <summary>
+    /// If the inventory is empty.
+    /// </summary>
+    public bool isEmpty
```

Comment:
not sure how crucial this is but the naming here is lowercase i from before instead of uppercase

Reply:
previously it was a function vs now its a computed property, which is why i swapped it. technically its a public property so it should be capitalized in C# for that reason, but that convention hasn't been followed anywhere else across the project so i made it lowercase for consistency rather than redoing every public variable across the entire project (i also just dont like that convention anyways)